en/firefox/welcome/page23.ftl

```diff
@@ -0,0 +1,19 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/welcome/23/
+
+welcome-page23-old-version = You’re on an older version of { -brand-name-firefox }
+
+welcome-page23-update-before = Update before March 14, 2025 to keep using { -brand-name-firefox }
+
+# Variables:
+#   $sumo (url) - link to https://support.mozilla.org/kb/root-certificate-expiration
+welcome-page23-your-version-of-firefox = <strong>Your version of { -brand-name-firefox } will stop working soon.</strong> A root certificate will expire, streaming video, add-ons, and other features to stop working. <a { $sumo }>Read more about this change</a>
```

Comment:
This string is missing a "causing". I'm going to fix it, since Pontoon hasn't synced yet, but the same fix needs to happen in Bedrock (cc @peiying2 and @craigcook)